ui/DesignSystem/Component/SourceBrowser/RevisionSelector.svelte

```diff
@@ -1,31 +1,168 @@
 <script>
-  import { gql } from "apollo-boost";
-  import { getContext } from "svelte";
-  import { getClient, query } from "svelte-apollo";
-
-  import { Input } from "../../Primitive";
-  import { revisionStore } from "../../../store/sourceBrowser.js";
-  import { HIDDEN_BRANCHES } from "../../../config.js";
-
-  const ALL_REVISIONS = gql`
-    query($projectId: ID!) {
-      tags(id: $projectId)
-      branches(id: $projectId)
-    }
-  `;
-
-  const allRevisions = query(getClient(), {
-    query: ALL_REVISIONS,
-    variables: { projectId: getContext("projectId") }
-  });
+  /* import { gql } from "apollo-boost"; */
+  /* import { getContext } from "svelte"; */
+  /* import { getClient, query } from "svelte-apollo"; */
+  import ClickOutside from "svelte-click-outside";
```

Comment:
I think we just removed this dependency - the same functionality can be achieved with `<svelte:window onClick={hideDropdown} />`